packages/ui/cypress/tests/watched-acccounts/watched-accounts.cy.ts

```diff
@@ -0,0 +1,62 @@
+import { addresses } from '../../fixtures/accounts'
+import { localHost } from '../../fixtures/landingData'
+import { landingPage } from '../../support/page-objects/landingPage'
+import { settingsPage } from '../../support/page-objects/settingsPage'
+
+describe('Watched Accounts', () => {
+  it('can add an account to the watch list', () => {
+    cy.visit(localHost)
+    landingPage.watchAccountButton().click()
+    settingsPage.accountAddressInput().type(`${addresses.Alice}{enter}`)
+    // autocomplete on the input seems to affect reliability so we wait a bit
+    cy.wait(1000)
+    settingsPage.accountNameInput().type('{selectall}{del}Alice')
+    settingsPage.addButton().click()
+    settingsPage.accountContainer().within(() => {
+      settingsPage.accountIcon().should('be.visible')
+      settingsPage.accountAddressLabel().should('be.visible')
+      settingsPage.accountNameLabel().should('be.visible')
+      settingsPage.accountDeleteButton().should('be.visible')
+    })
+  })
+
+  it('can remove an account from the watch list', () => {
+    cy.window().then((win) => {
+      win.localStorage.setItem('multix.watchedAccount', `["${addresses.AliceSS58}"]`)
+    })
+    cy.visit(localHost)
+    landingPage.watchAccountButton().click()
+    settingsPage.accountContainer().within(() => {
+      settingsPage.accountDeleteButton().click()
+      settingsPage.accountIcon().should('not.exist')
+      settingsPage.accountAddressLabel().should('not.exist')
+    })
+    settingsPage.accountContainer().should('have.length', 0)
+  })
+
+  it('can see error when attemping to add same address more than once', () => {
+    // stub existing watched account
+    cy.window().then((win) => {
+      win.localStorage.setItem('multix.watchedAccount', `["${addresses.AliceSS58}"]`)
+    })
+    cy.visit(localHost)
+    landingPage.watchAccountButton().click()
+    // attempt to add the same account again
+    settingsPage.accountAddressInput().type(`${addresses.Alice}{enter}`)
+    cy.wait(1000)
+    settingsPage.accountNameInput().type('{selectall}{del}Alice')
+    settingsPage.addButton().click()
+    settingsPage.errorLabel().should('be.visible')
```

Comment:
We should be specific on this one, and check for the error message. It failed for me for "invalid address", although the check is for something different.

Reply:
@Tbaut I was going to....but then remembered back to when I did that on another project and we said we didn't want to check the actual string 😄. I will take a look and add this in

Reply:
Updated and improved now, check the string and also that the add button is disabled